### DISCUSSION TITLE
Adding Days in Spanish to drop down menu

### DISCUSSION THREAD

**Discussion by WebTri2022** [code4recovery/12-step-meeting-list#878] [Q&A] [answered]
Hi,
When I post new meetings we can only choose days in English. 
For Spanish meetings, I’d like to put the days in Spanish. Is there a way to do this?

**Comment by 97eighty** [comment]
@WebTri2022 Hi, To better understand your question I'd like to see how you're using the plugin. What is the site URL for your site?

**Comment by WebTri2022** [comment]
Thank you for your response. https://trivalleyaa.org/

**Reply by 97eighty** [reply]
@WebTri2022 Hi, As I thought your site is in English supporting Spanish speaking meetings. The language features we have in place would change all the strings to Spanish, which I do not believe you want. I'm thinking that a possible feature would be to display the Spanish meetings using Spanish strings (i.e.  Sunday becomes Domingo. Perhaps @joshreisner, who maintains TSML-UI, has an opinion? My fellowship supports multiple languages too and, at first glance, this looks like a good feature to pursue. Adding a language parameter to the meeting URL may be a start. Be patient, these type of changes may take some time.

**Reply by joshreisner** [reply]
Looks like they are using TSML on their site. But TSML UI may be a solution; it shows Spanish (or French) if the user's browser is set to that language.

<img width="1261" alt="Screen Shot 2022-07-31 at 7 19 00 AM" src="https://user-images.githubusercontent.com/1551689/182030709-57b33f8f-d665-4bd5-97ea-2dbf115bcd12.png">

Try changing your system language, or getting a locale switcher plugin, and navigate to aasanjose.org/meetings. While the site unfortunately stays in English, the meeting finder switches to the language you have selected.

**Comment by WebTri2022** [comment]
Hello Josh,

Thank you for the help and information. I looked up the plug in TSML-UI and
made sure both plugins were installed. While your website is showing up in
Spanish when I change my language settings, my website still is not.

Please advise. Feel free to email me directly ***@***.***

Thank you for your service,
Michelle

**Comment by joshreisner** [comment, marked as answer]
Hi @WebTri2022 , doesn't look like TSML UI is enabled on [your website](https://trivalleyaa.org/meetings/) yet. TSML UI is a setting at Meetings -> Import & Settings -> Settings -> User Interface Display. (It's not a plugin, if you installed something new you can unstall it.)

Just to note, the "legacy UI" TSML also supports Spanish. You would enable that by installing a translation plugin like [Polylang](https://wordpress.org/plugins/polylang/). This might be a job for a web servant, if you have one, who could test it out on a demo site before enabling it on your main website. Once enabled, it would allow Spanish-language visitors to experience a fully-translated website experience.

**Reply by WebTri2022** [reply]
Hi,

I tried adding the Polylang plugin and it sent the whole test site into dissarray. Any suggestions about how to fix this or what 
I did wrong? 

Thank you,
Michelle

**Reply by joshreisner** [reply]
Glad you were using a test site! Polylang can be pretty involved, the expectation is that you would essentially be building a second website for Spanish. At any rate, it's beyond the ken of this discussion area.

How about Weglot? https://wordpress.org/plugins/weglot/ it puts a floating translation menu at the bottom of the page. it's used here: https://aasanjose.org/

Or you could use TSML UI of course, this would just translate the meeting finder section

**Reply by WebTri2022** [reply]
So Weglot is working... awesome! Now we're over the 10,000 word limit... do you pay for the yearly plan for 3 languages and 50,000 words? Why or why not?

**Reply by joshreisner** [reply]
i’m not sure, but you might contact the san jose intergroup and ask what they’re doing

**Comment by WebTri2022** [comment]
Weglot worked! Thank you so much for responding so many times!
